spec/unit/remote_system_spec.rb

```diff
@@ -18,7 +18,15 @@
 require_relative "spec_helper"
 
 describe RemoteSystem do
-  let(:remote_system) { RemoteSystem.new("remotehost") }
+  let(:rsys) { RemoteSystem.new("remotehost") }
```

Comment:
subject { RemoteSystem.new("remotehost", options) }

so you can always use subject and you only define options in the context that you need them like

let(:options_empty) { {} }
let(:options_ssh_port} { {ssh_port: 9999 } }